services/docker/docker-size.service.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const prettyBytes = require('pretty-bytes')
+const { anyInteger } = require('../validators')
+const { buildDockerUrl, getDockerHubUser } = require('./docker-helpers')
+const { BaseJsonService } = require('..')
+const { NotFound } = require('..')
+
+const buildSchema = Joi.object({
+  name: Joi.string(),
+  full_size: anyInteger,
```

Comment:
Should this be `nonNegativeInteger` instead?

Reply:
Yep that works better for the validation, should probably be adjusted in the other size badge which I based this from.